aea/skills/base.py

```diff
@@ -241,6 +242,10 @@ def __init__(
         self._configuration = configuration
         self._name = name
         self._context = skill_context
+        if len(kwargs) != 0:
```

Comment:
We give a warning if kwargs end up at the base class as this means they have not been retrieved in the derived class.